src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/filters/suppressionxpathsinglefilter/Example5.java

```diff
@@ -9,8 +10,10 @@
 </module>
 */
 
+package com.puppycrawl.tools.checkstyle.filters.suppressionxpathsinglefilter;
+
 // xdoc section -- start
-public interface TestClass {
-  public static final int CONSTANT1 = 1;  // OK
+public interface Example5 {
+  public static final int CONSTANT1 = 1; // 3 filtered violations
```

Comment:
please update code to make single violation ,

and please remove support for `X filtered violations`,
if we do it, we should have:
```
3 filtered violations above:
    ' blssssss'
    ' blaaaaa'
    ' ssssss'
``` 

but lets not do this in this PR, it is better to merge this PR sooner.

Reply:
similar activity and explanation https://github.com/checkstyle/checkstyle/pull/16286#issuecomment-2645741954